<launch>
    <node pkg="imageProcessing" name="detect_lines" type="detectLines.py" output="screen">
    </node>
    <node pkg="imageProcessing" name="edgdeDetect" type="edgeDetection.py" output="screen">
    </node>
    <node pkg="rqt_image_view" type="rqt_image_view" name="edge_detection" args="/image_edges"/>
    <node pkg="rqt_image_view" type="rqt_image_view" name="white_edges" args="/image_lines_white"/>
    <node pkg="rqt_image_view" type="rqt_image_view" name="yellow_edges" args="/image_lines_yellow"/>
    <node pkg="image_processing_hw" type="image_pub_all.py" name="image_pub_all" output="screen">
    </node>
</launch>